build.sbt

```diff
@@ -23,18 +25,21 @@ lazy val commonSettings = Seq(
     if (isSnapshot.value)
       Some("snapshots" at nexus + "content/repositories/snapshots")
     else
-      Some("releases"  at nexus + "service/local/staging/deploy/maven2")
+      Some("releases" at nexus + "service/local/staging/deploy/maven2")
   }
 
-  )
+)
 
 lazy val abandon = (project in file(".")).
   aggregate(base, cli, gui).
   dependsOn(base, cli, gui).
+  enablePlugins(BuildInfoPlugin).
   settings(commonSettings: _*).
   settings(
     name := "abandon",
-    fork in run := true
+    fork in run := true,
+    buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion, sbtVersion),
+    buildInfoPackage := "build.info"
```

Comment:
I guess it makes sense. I just want to note that `BuildInfo` is generated in the target folder.